tests/test_other.py

```diff
@@ -7774,12 +7774,16 @@ def test_output_eol(self):
           if '--separate-asm' in params: files += ['a.asm.js']
           if output_suffix == 'html': files += ['a.html']
           cmd = [PYTHON, EMCC, path_from_root('tests', 'hello_world.c'), '-o', 'a.' + output_suffix, '--output_eol', eol] + params
-          print(cmd)
```

Comment:
The convention has been in these tests to print out the test case that is being executed, in case there are multiple, so that it is easier to read which test case fails if something does regress. (i.e. the intent here has not been a 'leftover debug print'). Or was this a duplicate print somehow?